src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java

```diff
@@ -0,0 +1,151 @@
+package org.geoserver.geofence.rest;
```

Comment:
Hi all

I am about to finish GISP 120 and I put the security rest code in
gs-restconfig, hope this is ok

On Thu, May 14, 2015 at 4:33 PM, NielsCharlier <notifications@github.com>
wrote:

> In
> src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java
> <https://github.com/geoserver/geoserver/pull/1044#discussion_r30325318>:
>
> > @@ -0,0 +1,151 @@
> > +package org.geoserver.geofence.rest;
>
> Should there perhaps be a gs-rest-sec, and then any rest for the default
> security framework can be put there? And keep the geofence rest in this one
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoserver/geoserver/pull/1044/files#r30325318>.
>



-- 
DI Christian Mueller MSc (GIS), MSc (IT-Security)
OSS Open Source Solutions GmbH